src/webapi/window.rs

```diff
@@ -86,4 +88,17 @@ impl Window {
             ).into_reference_unchecked()
         }
     }
+
+    /// Sets a callback in requestAnimationFrame to call on next frame
+    /// Callback accepts an f64 as an argument
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)
+    pub fn request_animation_frame< F: FnMut(f64) + 'static>( &self, callback: F) -> RequestAnimationFrameHandle {
+        let id : u32 = js!{
```

Comment:
Why `u32`? This should probably be `i32`, which is the "native" integer type in the JS world.

Reply:
I think the id is always a positive integer, but that might be totally imagined by me. To make things line up easier I changed it to `i32`